apps/swirl-docs/src/components/Search/Algolia.tsx

```diff
@@ -0,0 +1,25 @@
+import { FunctionComponent } from "react";
+import algoliasearch from "algoliasearch/lite";
+import { InstantSearch } from "react-instantsearch-dom";
+import SearchBox from "./SearchBox";
+import Hits from "./Hits";
+import { ALGOLIA_INDEX } from "@swirl/lib/search";
+
+// Initialize the Algolia client
+const searchClient = algoliasearch(
+  process.env.NEXT_PUBLIC_ALGOLIA_APP_ID!!,
+  process.env.NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY!!
```

Comment:
**question:** Just to make sure. This is a public API key which is save to be used here?

Reply:
yes, based on the [tutorial](https://www.algolia.com/doc/guides/building-search-ui/getting-started/react/) at least :D